add snowpack on dapp staking BG

sirius651 opened the pull request:
**Pull Request Summary**

<img width="1320" alt="스크린샷 2022-12-16 오후 6 40 23" src="https://user-images.githubusercontent.com/1606820/208091609-de5d2fea-7151-4742-9388-61988ae25684.png">

for Xmas, inspired from https://codepen.io/alphardex/pen/dyPorwJ

**Check list**
- [ ] contains breaking changes
- [x] adds new feature
- [ ] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tested on mobile devices

Kahonnohak commented:
Great work!! It is something I was imagining so nice. Did not think we could have time!

Can we only have the snow in the hero? I can see some pop out on the right.

Please replace the logo with this and let's see how it looks:)
![astar-holiday-logo_FIN](https://user-images.githubusercontent.com/77480847/208126196-30a02ca8-aa66-420b-95c3-e69c01abc59f.png)

impelcrypto commented:
Amazing!

impelcrypto commented:
W 1700px~:  Could you add the snow on the left side (or remove it on the right side)?  
![image](https://user-images.githubusercontent.com/92044428/208139907-f71d0aaa-3506-4cc3-89fa-1448448f6678.png)

Kahonnohak commented:
@impelcrypto 
Oh i thought I'd rather remove the right area.

sirius651 commented:
@Kahonnohak @impelcrypto Changed logo & set max width for resolving above.

Kahonnohak commented:
Have you changed anything? The size of the logo is different. Snow is lovely! 

<img width="497" alt="スクリーンショット 2022-12-19 7 10 38" src="https://user-images.githubusercontent.com/77480847/208368189-1407c9ba-5b10-49c8-8e20-5354f17f173b.png">
<img width="458" alt="スクリーンショット 2022-12-19 7 10 52" src="https://user-images.githubusercontent.com/77480847/208368192-da80605e-2456-42fb-b494-b60a76e4308d.png">

sirius651 commented:
> Have you changed anything? The size of the logo is different. Snow is lovely!

yes I updated size of the logo more bigger. but I reverted it. please check @Kahonnohak

Kahonnohak reviewed (approved):
Looks great!

Let me ask the marketing team if it needs to be aligned when merging.
